source/guides/maintenance/cpu-performance.rst

```diff
@@ -58,63 +54,60 @@ or completely disabled with :command:`idle=poll`.
 
 .. note::
 
-   * :command:`processor.max_cstate=0` is changed to :command:`processor.max_cstate=1`
-     by the kernel to be a valid value.
+   *  :command:`processor.max_cstate=0` is changed to a valid value by the
+      kernel: :command:`processor.max_cstate=1`.
 
-   * :command:`intel_idle.max_cstate=0` disables the Intel Idle driver, not set
-     it to C-state 0.
+   *  :command:`intel_idle.max_cstate=0` disables the Intel Idle driver rather
+      than set it to C-state 0.
 
 .. _p-states-section:
 
 P-states (performance states)
 =============================
 
-P-states, also known as *Intel SpeedStep® technology* on Intel processors or
-*Cool'n'Quiet* on AMD processors, are states entered while the CPU is active and
-executing instructions.
-
-P-states aim to reduce power utilization by adjusting CPU clock frequency and
-voltages based on CPU demand.
-
-P-states can typically be limited or disabled in a system's firmware (UEFI/BIOS).
+The CPU can enter a P-state, also known as *Intel SpeedStep® technology* on
```

Comment:
Remove asterisks, please, in both cases. 

Revise to: "or with AMD* Cool'n'Quiet* technology [ ...]"